test/fuzz/mutable_visitor.cc

```diff
@@ -16,7 +16,7 @@ namespace {
 void traverseMessageWorkerExt(ProtoVisitor& visitor, Protobuf::Message& message,
                               std::vector<const Protobuf::Message*>& parents,
                               bool was_any_or_top_level, bool recurse_into_any,
-                              absl::string_view const& field_name) {
+                              const absl::string_view field_name) {
```

Comment:
You probably don't need the const. 
string_view  is exactly analogous to how a const char* . Thus,  marking string_view as const only affects whether the string_view object itself can be modified.

Reply:
Removed, thanks for the explanation.

Reply:
Thanks.

Reply:
This is similar to other primitives as well, so if you want to change the `const bool` in the other parameters feel free to do so.